### DISCUSSION TITLE
week 11 post

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#212] [General]
<img width="477" alt="Screenshot 2023-07-22 182929" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133051222/0719f9c8-c637-4d1d-93c2-6df86079fc4f">

Link to project: https://github.com/USU-ITLS-x265-23-summer/Skye404Project/tree/main

**Comment by kassyoveson** [comment]
You have made a lot of progress this week. You can save the image file in the same folder as your HTML file, and then specify the path to the image file in the src attribute of the <img> tag. For example, if the image file is named my_image.png, and it is saved in the same folder as the HTML file, then you would use the following code to insert the image into the HTML document: <img src="my_image.png" alt="My image"></img> I hope this helps. Your projects is looking good.

**Reply by Skye404** [reply]
I think I finally got the image figured out. Thanks! I'm going to work on adding the pdf resume now.

**Comment by CJ3315** [comment]
Great work! Keep it up!
